src/main/groovy/com/marklogic/gradle/task/MlcpTask.groovy

```diff
@@ -208,7 +208,11 @@ class MlcpTask extends JavaExec {
 		File logOutputFile = null
 		if (logOutputUri) {
 			println "Will write mlcp log output to URI: " + logOutputUri
-			logOutputFile = new File(getProject().getBuildDir(), "mlcp-log-output-" + System.currentTimeMillis() + ".txt")
+			def buildDir = getProject().getBuildDir()
```

Comment:
One tweak - as noted in the ticket, this is deprecated. Looks like this is what Gradle wants now - https://stackoverflow.com/questions/77564583/getting-rid-of-builddir-deprecation - so something like this:

    getProject().layout.buildDirectory.file("mlcp-log-output....")